src/main/java/com/kodilla/ecommercee/domain/User.java

```diff
@@ -0,0 +1,53 @@
+package com.kodilla.ecommercee.domain;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Entity
+@Table(name="users")
+public class User {
+
+    @Id
+    @GeneratedValue(strategy= GenerationType.AUTO)
+    private Long id;
+
+    @Column(name="firstName")
```

Comment:
```suggestion
    @Column(name="first_name")
```

Reply:
Column name in database should consist only of small letters and _  playing role of separator between words